src/main/java/org/breedinginsight/brapi/v2/services/BrAPITrialService.java

```diff
@@ -465,6 +468,20 @@ private Map<String, Object> createExportRow(
         row.put(ExperimentObservation.Columns.EXP_TYPE, experiment.getAdditionalInfo().getAsJsonObject().get(BrAPIAdditionalInfoFields.EXPERIMENT_TYPE).getAsString());
         row.put(ExperimentObservation.Columns.ENV, Utilities.removeProgramKeyAndUnknownAdditionalData(study.getStudyName(), program.getKey()));
         row.put(ExperimentObservation.Columns.ENV_LOCATION, Utilities.removeProgramKey(study.getLocationName(), program.getKey()));
+
+        // Lat, Long, Elevation
+        Coordinates coordinates = extractCoordinates(ou);
+        row.put( ExperimentObservation.Columns.LAT, coordinates==null? null : doubleToString(coordinates.getLat()) );
+        row.put( ExperimentObservation.Columns.LONG, coordinates==null? null : doubleToString(coordinates.getLon()) );
+        row.put( ExperimentObservation.Columns.ELEVATION, coordinates==null? null : doubleToString(coordinates.getAlt()) );
```

Comment:
Looks good! Refactoring using Optionals here might help the readability.
```suggestion
    // Lat, Long, Elevation
    Coordinates coordinates = extractCoordinates(ou);
    Optional.ofNullable(coordinates)
            .map(c -> doubleToString(c.getLat()))
            .ifPresent(lat -> row.put(ExperimentObservation.Columns.LAT, lat));
    Optional.ofNullable(coordinates)
            .map(c -> doubleToString(c.getLon()))
            .ifPresent(lon -> row.put(ExperimentObservation.Columns.LONG, lon));
    Optional.ofNullable(coordinates)
            .map(c -> doubleToString(c.getAlt()))
            .ifPresent(elevation -> row.put(ExperimentObservation.Columns.ELEVATION, elevation));
```